apps/vite/src/components/calendar/Today.tsx

```
@@ -1,55 +1,120 @@
-          src="https://img.icons8.com/color/48/000000/sunset.png"
-          alt="sunrise"
-          className="h-6 w-6"
-        />
-        <p className="text-sm text-gray-600">सूर्यास्त: १७:३९</p>
+Today.skeleton = (
+  <div className="flex flex-col rounded-lg">
+    <div className="flex items-center space-x-4">
+      <div className="animate-pulse rounded-lg bg-gray-200 text-center w-16 h-16 flex flex-col gap-1 items-center justify-center">
+        <div>
+          <div className="h-6 w-8 bg-gray-300 rounded-md"></div>
+          <div className="h-4 w-10 bg-gray-300 rounded-md mt-1"></div>
```

Biome

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)